jackson/src/main/kotlin/dev/codified/jackson/CodifiedJacksonModule.kt

```diff
@@ -0,0 +1,36 @@
+package dev.codified.jackson
+
+import com.fasterxml.jackson.core.Version
+import com.fasterxml.jackson.databind.Module
+import com.fasterxml.jackson.databind.module.SimpleDeserializers
+import com.fasterxml.jackson.databind.module.SimpleSerializers
+import pl.brightinventions.codified.Codified
+import pl.brightinventions.codified.enums.CodifiedEnum
+
+class CodifiedJacksonModule : Module() {
+    override fun version(): Version {
+        return Version.unknownVersion()
```

Comment:
Not a top priority, I guess, but do you think it would make sense to return something more useful here?

Reply:
Probably. However, I don't know how to read our Versions here, hence I left it for later 🙈